InfrastructureLayer/Settings/I_Settings.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+class I_Settings
+{
+public:
+    virtual ~I_Settings() {}
+    virtual QString ipAddress() const = 0;
+    virtual quint16 udpPort() const = 0;
+    virtual int forwardPeriod() const = 0;
+    virtual QString packetTitle() const = 0;
+    virtual QString exchangeName() const = 0;
+    virtual int numberOfCmus() const = 0;
```

Comment:
Lots of other code depends on the number of CMUs and MPPTs etc being the same, so this isn't really a configurable setting. Like this, it's kind of a trap for someone to break the code, unless our code can actually handle other numbers of cmus etc, this shouldn't be configurable.